src/metaform-react/MetaformTableFieldComponent.tsx

```diff
@@ -104,10 +104,14 @@ export const MetaformTableFieldComponent: React.FC<Props> = ({
     }
     
     return (
-      <Button style={{ cursor: "pointer" }} onClick={ onAddRowButtonClick }>
+      <SubmitFieldWrapper
```

Comment:
Bit confusing to use submitFieldWrapper here. Maybe make another styled component for addRowButton.